client/jetpack-cloud/sections/partner-portal/utils.ts

```diff
@@ -176,3 +182,13 @@ export function formatApiPartner( partner: APIPartner ): Partner {
 export function getProductTitle( product: string ): string {
 	return product.replace( /(?:Jetpack\s|[)(])/gi, '' );
 }
+
+export function selectProductOptions( families: APIProductFamily[] ): APIProductFamilyProduct[] {
+	return families.flatMap( ( family ) => family.products );
+}
+
+export function alphabeticallySortedProductOptions(
```

Comment:
Let's rename this to make it obvious it's a query selector function like the one above:
`selectAlphabeticallySortedProductOptions`